.unreleased/LLT-5319

```diff
@@ -0,0 +1 @@
+Bump crypto_box version and move to our ChaChaBox implementation
```

Comment:
I don't think this needs a changelog entry since apps don't need to care about this change

Reply:
Hmm maybe? There was a vulnerability found in one of the crates used by the old `crypto_box`, co I thought it is worth mentioning.

Reply:
That's a good point, but in that case I think the CVE that's fixed should be mentioned to make that more clear